grep-00XX-relicense-volk.md

```diff
@@ -0,0 +1,172 @@
+With a formal process and a clear vision laid out, we hope to be able to get everyone on board.
+
+Since VOLK is a library it should choose a fitting license.
+If we see greater adoption of VOLK in more projects, we are confident that we will receive more contributions.
+May it be bug fixes, new kernels or even contributions to core features.
+
+
+## Description
+
+In order to facilitate this relicensing effort, we need to got through several steps.
+
+
+### Notification
+
+We consider it to be very important to let every contributor know that the wish to relicense by some contributors exists.
+Based on the information we can gather through git commit names and email addresses as well as GitHub accounts, we notify every contributor of this GREP and ask them to resubmit their contributions.
+
+
+### The new license
+
+We propose [LGPLv3+](https://www.gnu.org/licenses/lgpl-3.0.html) for the new license.
+LGPLv3+ strikes a balance between permissiveness and obligation to provide modified source code.
+All modifications to VOLK must be made available under the same terms under which everyone can use VOLK.
+This should not require users to share code that uses VOLK unaltered.
+
+
+### All contributors need to re-submit their code under the new license
+
+Whether one submitted code under a CLA or with a DCO, every contributor must re-submit their code under the new license.
+A contributor may submit their code to any project, including VOLK under a new license, even if that code was submitted under a CLA.
+
+
+### The process
+
+Currently, we have around 65 contributors to VOLK.
+In order to relicense VOLK, every contributor must re-submit their code under the new license.
+During the transition period, we want to be able to accept new contributions.
+
+1. We notify every contributor that we want to relicense VOLK.
+2. New contributions are required to be submitted under the new license.
+3. Contributors are kindly asked to resubmit their contributions.
+4. After all contributors have resubmitted their contributions, we have a new major release and continue VOLK development under the new license.
+
+The new major release updates all license headers to state the new license.
+The transition process is executed in-place, i.e. in the current VOLK repository, to not add any confusion.
+
+#### How to resubmit contributions
+We decided to adopt the following process to resubmit contributions.
+
+1. We collect all resubmissions in a file inside the VOLK repository.
+2. All contributors are kindly asked to add themselves to this list.
+3. By adding themself to the list, a contributor resubmits their code to VOLK under the new license.
+4. Contributors have several options to add themselves to the list
+    - Open a PR against the VOLK repository and add themself to the list.
+    - Write an email to the current VOLK maintainers that clearly states the required author information. 
```

Comment:
```suggestion
    - Open a PR against the VOLK repository and add themself to the list.
    - Write an email to the current VOLK maintainers that clearly states the required author information.
    - Add a line saying they agree to a public GitHub issue.
    - Sign a paper form and make it available to us. This is generally cumbersome, but can be a useful method at events where people are available in-person.
```

Reply:
Rationale: We should make it as easily as humanly possible to resubmit. If we only get a developers attention for 30 seconds, those need to suffice.

Reply:
agreed with @mbr0wn here

Reply:
I agree. I just added it. We'd need to figure out who'd be in charge of storing them though. Would this be a SETI thing?